scripts/asm-installer/install_asm

```diff
@@ -496,7 +502,11 @@ OPTIONS:
                                       omitted if not installing optional
                                       features. To add multiple files, specify
                                       them with multiple options one at a time.
-
+  --ca_name|--ca-name    <CA NAME>    Required only if --ca option is private_ca.
+                                      Name of the ca in the GCP CAS service used to
+                                      sign certificates in the format
+                                      'projects/project_name/locations/ \
+                                       ca_region/certificateAuthorities/ca_name'
```

Comment:
Why does this entire thing need to be entered by the user? We should know $PROJECT_ID and $CA_NAME already. Does Private CA support clusters in different regions?